pandas/tests/indexing/test_iloc.py

```diff
@@ -35,28 +35,12 @@
 
 
 class TestiLoc(Base):
-    def test_iloc_getitem_int(self):
+    @pytest.mark.parametrize("key", [2, -1, [0, 1, 2]])
+    def test_iloc_getitem_int(self, key):
```

Comment:
rename since it is no longer just int keys?

Reply:
Done